### ISSUE TITLE
"okra generate" fails with "No SSL or TLS support compiled into Conduit" on M1 mac

### ISSUE THREAD

**Issue by gridbugs** [MagnusS/okra#110]
I get this error on MacOS and dockerized Ubuntu running on an M1 mac:
```
$ okra generate --week=36
okra: internal error, uncaught exception:
      Failure("No SSL or TLS support compiled into Conduit")
```

**Comment by smorimoto** [comment]
From README.md

> If you get an HTTPS error while downloading the Github activity, TLS support is probably not compiled into cohttp/conduit. opam install tls should fix this.

**Comment by MagnusS** [comment]
This is documented and caused by an upstream dependency, but as this seems to be a very common issue for new users I added a temporary fix in #112
